queries/commits_query.py

```diff
@@ -41,5 +46,8 @@ def commits_query(dbmc, repo):
     dbm.load_pconfig(dbmc)
     df_commits = dbm.run_query(query_string)
 
+    # massage SQL timestampdz to Posix timestamp int64- much smaller than string or datetime object.
+    df_commits["date"] = pd.to_datetime(df_commits["date"], utc=True).map(pd.Timestamp.timestamp).astype("int64")
```

Comment:
when you get the data on the graph generation side can you use it in the same way as before?

Reply:
yep, you just use pd.to_datetime() on the column, you can see how it works in the callback